src/Altinn.App.Api/Controllers/Attributes/JsonSettingsNameAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Altinn.App.Api.Controllers.Attributes;
+
+[AttributeUsage(AttributeTargets.Class)]
+internal class JsonSettingsNameAttribute : Attribute, IFilterMetadata
+{
+    internal JsonSettingsNameAttribute(string name)
+    {
+        Name = name;
+    }
+
+    internal string Name { get; }
+}
+
+internal static class JsonSettingNames
+{
+    internal const string ALTINN_API = "AltinnApi";
```

Comment:
I thought the convention for consts was just PascalCase now, do you get any IDE hints/warnings?

Reply:
VsCode with C#, C# devkit, and C# Utilities extensions seems to think this is OK. But I see that Microsoft disagrees: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions..

Reply:
I thought there was a rule touching on this in .editorconfig, maybe the EditorConfig ext is needed as well